packages/stores/src/queries-external/store.ts

```diff
@@ -29,3 +38,39 @@ export class ObservableQueryExternalBase<
     super(kvStore, instance, urlPath);
   }
 }
+
+export class ObservableQueryIbcStatus extends ObservableQueryExternalBase<IbcStatus> {
```

Comment:
I actually just added an external query on another branch myself. Did you put the IBC status class defs here because of a circular dependency issue? I'd recommend trying to move it into an index.ts file in the `/ibc-status/` folder.

If you run into an issue, you can leave this here. It can be moved later. My solution was to create a `base.ts` file here and moving the base object there.

Reply:
Yup gotcha, that would be consistent with the pattern set by `pool-fees` looks like